### ISSUE TITLE
Fix false negatives for selectors containing comments in selector-descendant-combinator-no-non-space

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#4599]
> Clearly describe the bug

Fix a limitation in `selector-descendant-combinator-no-non-space`, introduced in https://github.com/stylelint/stylelint/pull/4595/, where selectors containing comments are ignored.

> Which rule, if any, is the bug related to?

`selector-descendant-combinator-no-non-space`

> What CSS is needed to reproduce the bug?

```css
.foo >    /*comment*/    .bar {}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "selector-descendant-combinator-no-non-space": true
  }
}
```

> Which version of stylelint are you using?

13.2.0

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

Warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

No warnings were flagged.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and fix this bug.
